lib/admin/WP_Auth0_Admin_Appearance.php

```diff
@@ -305,6 +306,17 @@ public function basic_validation( $old_options, $input ) {
 			self::add_validation_error( __( 'The Lock JS CDN URL used is not a valid URL.', 'wp-auth0' ) );
 		}
 
+		$input['lock_connections'] = isset( $input['lock_connections'] ) ?
```

Comment:
Same here, just moving the settings to the right place. Will address validation here in a future PR